packages/store/src/schematics/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "ngxs-schematics",
+  "version": "0.0.0",
+  "description": "A Ngxs schematics",
+  "scripts": {
+    "build": "tsc -p tsconfig.json",
+    "test": "npm run build && jasmine **/*_spec.js"
+  },
+  "keywords": [
+    "schematics"
+  ],
+  "author": "Mikael Morlund",
+  "license": "MIT",
+  "schematics": "./src/collection.json",
+  "dependencies": {
+    "@angular-devkit/core": "^0.5.4",
+    "@angular-devkit/schematics": "^0.5.4",
+    "@types/jasmine": "^2.6.0",
+    "@types/node": "^8.0.31",
+    "jasmine": "^2.8.0",
+    "typescript": "^2.5.2"
+  }
+}
```

Comment:
do we need a separate package.json for this? or can it be in the root for this module

Reply:
Yes.